docs/prover/approx/hashing.md

```diff
@@ -0,0 +1,182 @@
+Modeling of Hashing in CVT
+===========
+
+
+## Introduction
+
+The Keccak hash function is used heavily by Solidity smart contracts. 
+Most prominently, all unbounded data structures in storage (arrays, mappings) receive their storage addresses as values of the Keccak function.
```

Comment:
Maybe worth mentioning here or somewhere that this approximation is also used when users call the solidity keccak functions (assuming that that's correct).

Reply:
right, done (further down there are also snippets using the other ways (direct call from solidity and inline assembly)